### PR TITLE
Milestone 2:

### PR DESCRIPTION
**Pull request by blueberry1312**
Added section "My recent works"
Fixed style problems
Added more svg files.
Added About me.
Added Categories.
Added more svg files.

### CODE REVIEW

**Review by frank1738 — COMMENTED** [review]
Hi @blueberry1312 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project whose readme is missing some important information. Kindly edit the `key features` and `future features`  with the project details for it to be professional

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html [hunk 1]
```diff
@@ -41,5 +41,193 @@ <h2>
                 </button>
             </div>
         </section>
+        <section>
+            <h1 class="work-title">My Recent Works</h1>
+            <div class="indicator">
```
Comment (ghost) [inline index.html:46]: - [ ] Good work on your project cards but I would highly recommend that you position them in the center of your mobile view

image

### Checkout the original design

image

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @blueberry1312  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

✅ Correct use of Github workflow
✅ No linter error


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by pinkmoon25 — APPROVED** [review]
Hi @blueberry1312 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linter checks :heavy_check_mark: 
Github flow :heavy_check_mark: 

## Optional suggestions

- [ ] please see there is a Horizontal Overflow :point_down: 
image

Please try to fix this by adjusting the overflow property :point_up: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**